client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -204,6 +204,10 @@ if(NOT MSVC)
    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -ggdb")
 endif()
 
+# "no-unused-private-field" is added since in most of cases it is false positive
+list(APPEND KAA_COMPILE_OPTIONS
+                -Wno-unused-private-field)
```

Comment:
I understood. I tried to tell that I do not see the reason :) I cannot find any place in DefaultLogger.cpp to put usage of private field 'context_' from ProfileManager.hpp